microsite/data/plugins/facets-cloud.yaml

```
@@ -0,0 +1,11 @@
+title: Facets.cloud Platform
```

I think you forgot the YAML document separator:
```suggestion
---
title: Facets.cloud Platform
```